src/components/network-map-tab.js

```diff
@@ -561,15 +561,30 @@ export const NetworkMapTab = ({
 
         dispatch(resetMapReloaded());
 
+        const currentNodeAtReloadCalling = currentNodeRef.current;
+
         return mapEquipments
             .reloadImpactedSubstationsEquipments(
                 studyUuid,
                 currentNode,
-                updatedSubstationsToSend,
-                setUpdatedLines
+                updatedSubstationsToSend
             )
-            .finally(() => {
-                setWaitingLoadData(false);
+            .then((results) => {
```

Comment:
this wait for both to be done, we want to update things as soon as the request finished